engine/execution/provider/engine.go

```diff
@@ -224,10 +232,8 @@ func (e *Engine) onChunkDataRequest(
 				Logger()
 		}
 
-		lg.Debug().Msg("chunk data pack request successfully replied")
+		lg.Info().Msg("chunk data pack request successfully replied")
```

Comment:
Is this really `Info()` I would imagine this is called a lot and would clog the logs

Reply:
Missing a chunk data pack can cause a sealing halt and it is really hard to track it down currently. However, I am developing some telemetry and soon we can switch it back to debug.